e2e-tests/insurance/cypress/e2e/journeys/export-contract/about-goods-or-services/about-goods-or-services.spec.js

```diff
@@ -72,20 +72,20 @@ context(
         const fieldId = DESCRIPTION;
         const field = aboutGoodsOrServicesPage[fieldId];
 
-        const fieldStrings = FIELDS.ABOUT_GOODS_OR_SERVICES[fieldId];
+        const { HINT, LABEL, MAXIMUM } = FIELDS.ABOUT_GOODS_OR_SERVICES[fieldId];
 
-        cy.checkText(field.hint.intro(), fieldStrings.HINT.INTRO);
+        cy.checkText(fieldSelector(fieldId).hintIntro(), HINT.INTRO);
 
-        cy.checkText(field.hint.list.item1(), fieldStrings.HINT.LIST[0]);
-        cy.checkText(field.hint.list.item2(), fieldStrings.HINT.LIST[1]);
-        cy.checkText(field.hint.list.item3(), fieldStrings.HINT.LIST[2]);
+        cy.checkText(field.hint.list.item1(), HINT.LIST[0]);
```

Comment:
could we dereference hint or list here from field?

Reply:
Updated all instances of `field.hint.list` to be `hint.list`